vector/src/main/res/drawable/ic_fab_add_by_mxid.xml

```diff
@@ -0,0 +1,30 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
This file will need to be replaced as it's just a placeholder for an actual icon. Perhaps a magnifying glass for search?